README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
-| Credential Evaluation  | ✅         | ✅     | ❌   | ❌    |
-| [Predicates](https://identity.foundation/presentation-exchange/spec/v2.0.0/#predicate-feature)                           | ✅         | ✅     | ❌   | ❌    |
-| [Relational Constraints](https://identity.foundation/presentation-exchange/spec/v2.0.0/#relational-constraint-feature)   | ✅         | ❌     | ❌   | ❌    |
-| [Credential Status](https://identity.foundation/presentation-exchange/spec/v2.0.0/#credential-status-constraint-feature) | ❌         | ❌     | ❌   | ❌    |
+| Feature                  |
+| ------------------------ |
+| Data model               | 
+| Signing and verification | 
+
+#### [Status List 2021](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/)
+
+For usage with the [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model).
+
+| Feature                                | 
+| -------------------------------------- | 
+| Data model                             |
+| Status checking                        |
+| Status setting                         |
+| Signing and verification with `vc-jwt` |
+
+#### [Bitstring Status List](https://www.w3.org/TR/vc-bitstring-status-list/)
+
+For usage with the [W3C Verifiable Credential Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/).
+
+| Feature                                      | 
+| -------------------------------------------- | 
+| Data model                                   |
+| Status checking                              |
+| Status setting                               |
+| Signing and verification with `vc-jose-cose` |
+
+
+#### [VC JSON Schema](https://www.w3.org/TR/vc-json-schema/)
+
+For usage with both the [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model) and the [W3C Verifiable Credential Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/).
+
+| Feature                                 |
+| --------------------------------------- | 
+| Creation and validation of `JsonSchema` |
+| Creation and validation of `JsonSchemaCredential` using `vc-jwt` with the VCDM v1.1 |
+| Creation and validation of `JsonSchemaCredential` using `vc-jose-cose` with the VCDM v2.0 |
+
+#### [Presentation Exchange v2.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
+
+| Feature                | 
+| ---------------------- | 
+| Data model             | 
+| Validation             |
+| Credential Evaluation using the VCDM v1.1 (both `vc-jwt` and `vp-jwt`), the VCDM v2.0 using `vc-jose-cose`, and SD-JWT   |
+| [Predicates](https://identity.foundation/presentation-exchange/spec/v2.0.0/#predicate-feature)                           |
+| [Relational Constraints](https://identity.foundation/presentation-exchange/spec/v2.0.0/#relational-constraint-feature)   |
+| [Credential Status](https://identity.foundation/presentation-exchange/spec/v2.0.0/#credential-status-constraint-feature) |
```

Comment:
good to know, I dont think we support this right now

Reply:
since status in our KCCs has been descoped this can probably be left out for now, though will be useful going forward